packages/gitmoji-changelog-markdown/src/index.js

```diff
@@ -103,6 +103,22 @@ function matchVersionBreakpoint(tested, version = '.*') {
   return regex.test(tested)
 }
 
+function getLatestVersion(markdownFile) {
+  let markdownContent
+
+  try {
+    markdownContent = fs.readFileSync(markdownFile, 'utf-8')
+  } catch (err) {
+    return null
+  }
+
+  const result = markdownContent.match(/<a name="([\w.]+?)"><\/a>/)
+
+  if (!result) return null
+
+  return `v${result[1]}`
```

Comment:
Are we sure that if `result` is not null you will always have an array with at least 2 entries. Maybe this could be checked.

Reply:
Sure this is documented [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/match#Return_value) 👍